docker/docker-compose.yml

```diff
@@ -4,33 +4,33 @@ services:
   vault_server:
     networks:
       - develop
-    links:
-      - "ganache"
     image: "cypherhat/vault-ethereum:latest"
     ports:
       - "9200:9200"
     volumes:
-      - "./config:/home/vault/config:rw"
+      - "./config:/home/vault/config:rw" #inside the container the root.crt file is available at /home/vault/config/root.crt
     entrypoint: /home/vault/config/entrypoint.sh
   ganache:
     networks:
       - develop
-    image: trufflesuite/ganache-cli:latest
+    image: trufflesuite/ganache:latest
+    platform: linux/arm64 # Explicitly use ARM64 for M3 compatibility
```

Comment:
provide in the comment the value for other platforms and describe in the README this point.